### ISSUE TITLE
Seeing runtime error "storage.getStore(...).addError is not a function"

### ISSUE THREAD

**Issue by dmattia** [DataDog/dd-trace-js#1740]
**Describe the bug**
I bumped up our dd-trace version from `^0.35.0` to `2.0.0-appsec-beta.4`, made some slight adjustments to how we initialize the tracer for the new initialization options, and our server began crashing with this error.

<img width="803" alt="Screen Shot 2022-01-06 at 2 26 13 PM" src="https://user-images.githubusercontent.com/8922077/148447235-b1f55112-f0ea-4b71-a806-75e128601403.png">

There is a bit of a stack trace in that image showing where the issue might be coming from. If it helps, our initialization logic now looks like:

```typescript
    tracer.init({
      env: options.DD_ENV,
      debug: options.DD_TRACE_DEBUG,
      ingestion: {
        sampleRate: options.DD_APM_ANALYTICS ? 1.0 : 0.0,
      },
      logInjection: options.DD_APM_LOG_INJECTION,
      runtimeMetrics: options.DD_APM_RUNTIME_METRICS,
      hostname: options.DD_HOST,
      dogstatsd: {
        port: options.DD_STATSD_PORT,
      },
      port: options.DD_APM_PORT,
    });
```

**Environment**

* **Operation system:** linux
* **Node version:** 14.17.3
* **Tracer version:** 2.0.0-appsec-beta.4
* **Agent version:** `datadog:latest`

**Comment by bengl** [comment]
Hi @dmattia.

Thanks for finding this! The fix is #1741 and it will most likely make it into the next 1.x release, pending review.

**Comment by dmattia** [comment]
@bengl Wonderful, thank you!

**Comment by simon-id** [comment]
@dmattia I have released the fix under the version `2.0.0-appsec-beta.5`. I assume you use AppSec so you can go ahead and update to this version!
